test/e2e/features/e2e.feature

```diff
@@ -0,0 +1,32 @@
+Feature: VxFlex OS CSI interface
+  As a consumer of the CSI interface
+  I want to run a system test
+  So that I know the service functions correctly.
+
+  @zone
+  Scenario: Create zone volume through k8s
+    Given a VxFlexOS service
+    And verify driver is configured and running correctly
+    And verify that the node and storage class zone label is <zoneLabelKey>
+    And verify that the secret <secret> in namespace <namespace> is set for zoning
+    Then create zone volume and pod in <location>
+    And check pods to be running on desired zones
+    Then delete zone volume and pod in <location>
+    Examples:
+      | zoneLabelKey                    | secret            | namespace  | location    |
+      | "zone.csi-vxflexos.dellemc.com" | "vxflexos-config" | "vxflexos" | "zone-wait" |
```

Comment:
Can we add an additional case here for a custom zone label?

Reply:
Yup, I was waiting for the other PR to add anything related to zoneLabelKey.

Reply:
Removed `zoneLabelKey` argument. E2E assumes that the driver is running and setup correctly so we are now retrieving the zone label from the secret and ensure that the label key is consistent across the nodes and storage class.